custom_components/hacs/base.py

```diff
@@ -674,12 +674,12 @@ async def startup_tasks(self, _=None) -> None:
             async_track_time_interval(self.hass, self.async_check_rate_limit, timedelta(minutes=5))
         )
         self.recurring_tasks.append(
-            async_track_time_interval(self.hass, self.async_process_queue, timedelta(minutes=10))
+            async_track_time_interval(self.hass, self.async_process_queue, timedelta(minutes=30))
```

Comment:
This does not do much, really; the queue is executed fully each time unless a rate limit is hit, and from the next version, that should not really be possible as most data is no longer fetched from GitHub.

In most cases, this queue is empty, so at best, it just adds 20 min extra to the current 48h/96h intervals.